sdk/eventhub/Azure.Messaging.EventHubs/stress/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: eventhub-net-stress-test
+version: 0.1.1
+description: Stress tests for Event Hubs for .NET
+
+dependencies:
+- name: stress-test-addons
+  version: 0.1.13
+  repository: https://stresstestcharts.blob.core.windows.net/helm/
+
+annotations:
+  stressTest: 'true'
+  namespace: 'net'
+  dockerbuilddir: '../../../..'
+  dockerfile: './Dockerfile'
```

Comment:
@ckairen I'm also realizing here that we have no easy way to docker build without deploying when adding the custom build dirs.

Reply:
Well taking the publish image code out as a separate script is not too hard, also helps decrease the size of deployment script which I'm sure will still grow. Do you think it's better to do it that way?

Reply:
I wonder if having a higher level type CLI/script for working with stress tests could be useful. E.g. `stress build`, `stress start`, `stress deploy`, `stress debug` etc. etc.

Reply:
@jsquire right now the script automation basically assumes you're doing everything: logging in, building and pushing images, and deploying the stress tests to the cluster. You can disable some of this functionality via script parameters but it doesn't apply to everything and isn't optimized for all dev workflows. I think that it would be better to re-think the UX of the stress command line tools.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/3193

Reply:
ahhhh....  Got it!